src/allotropy/allotrope/schemas/light-obscuration/REC/2021/12/light-obscuration.embed.schema.json

```diff
@@ -0,0 +1,1342 @@
+              "$asm.lookup-property": "http://www.w3.org/2004/02/skos/core#prefLabel"
+            }
+          }
+        }
+      ]
+    },
+    "equipment serial number": {
+      "$asm.property-class": "http://purl.allotrope.org/ontologies/result#AFR_0001119",
+      "$asm.pattern": "value datum",
+      "$asm.type": "http://www.w3.org/2001/XMLSchema#string",
+      "oneOf": [
+        { "type": "string" },
+        {
+          "type": "object",
+          "required": ["@type", "value"],
+          "properties": {
+            "value": { "type": "string" },
+            "@type": {
+              "type": "string",
+              "$asm.type": "http://www.w3.org/2000/01/rdf-schema#Class",
+              "$asm.lookup-property": "http://www.w3.org/2004/02/skos/core#prefLabel"
+            }
+          }
+        }
+      ]
+    },
+    "detector model number": {
+      "$asm.property-class": "http://purl.allotrope.org/ontologies/result#AFR_0002489",
+      "$asm.pattern": "value datum",
+      "$asm.type": "http://www.w3.org/2001/XMLSchema#string",
+      "oneOf": [
+        { "type": "string" },
+        {
+          "type": "object",
+          "required": ["@type", "value"],
+          "properties": {
+            "value": { "type": "string" },
+            "@type": {
+              "type": "string",
+              "$asm.type": "http://www.w3.org/2000/01/rdf-schema#Class",
+              "$asm.lookup-property": "http://www.w3.org/2004/02/skos/core#prefLabel"
+            }
+          }
+        }
+      ]
+    },
+    "model number": {
+      "$asm.property-class": "http://purl.obolibrary.org/obo/IAO_0000017",
+      "$asm.pattern": "value datum",
+      "$asm.type": "http://www.w3.org/2001/XMLSchema#string",
+      "oneOf": [
+        { "type": "string" },
+        {
+          "type": "object",
+          "required": ["@type", "value"],
+          "properties": {
+            "value": { "type": "string" },
+            "@type": {
+              "type": "string",
+              "$asm.type": "http://www.w3.org/2000/01/rdf-schema#Class",
+              "$asm.lookup-property": "http://www.w3.org/2004/02/skos/core#prefLabel"
+            }
+          }
+        }
+      ]
+    },
+    "measurement time": {
+      "$asm.property-class": "http://purl.allotrope.org/ontologies/result#AFR_0000952",
+      "$asm.pattern": "value datum",
+      "$asm.type": "http://www.w3.org/2001/XMLSchema#dateTime",
+      "oneOf": [
+        { "type": "string", "format": "date-time" },
+        {
+          "type": "object",
+          "required": ["@type", "value"],
+          "properties": {
+            "value": { "type": "string", "format": "date-time" },
+            "@type": {
+              "type": "string",
+              "$asm.type": "http://www.w3.org/2000/01/rdf-schema#Class",
+              "$asm.lookup-property": "http://www.w3.org/2004/02/skos/core#prefLabel"
+            }
+          }
+        }
+      ]
+    }
+  },
+  "$defs": {
```

Comment:
Assuming the defs match (they should) yes.

The idea here is that while we need "embeded" schemas because our model generator code can't handle following links to other references at this time, the point of the shared code in the normal schema is that we shouldn't have to duplicate all this code in every schema.